src/e2e/java/teammates/e2e/util/BackDoor.java

```diff
@@ -215,8 +215,10 @@ public static String removeAndRestoreDataBundle(DataBundle dataBundle) {
         removeDataBundle(dataBundle);
         ResponseBodyAndCode putRequestOutput =
                 executePostRequest(Const.ResourceURIs.DATABUNDLE, null, JsonUtils.toJson(dataBundle));
-        return putRequestOutput.responseCode == HttpStatus.SC_OK
-                ? Const.StatusCodes.BACKDOOR_STATUS_SUCCESS : Const.StatusCodes.BACKDOOR_STATUS_FAILURE;
+        if (putRequestOutput.responseCode == HttpStatus.SC_OK) {
+            return putRequestOutput.responseBody;
```

Comment:
The function return two different things (json response or `BACKDOOR_STATUS_FAILURE `) with different meanings which result in the hack in `https://github.com/TEAMMATES/teammates/pull/9967/files#diff-6509e6ed4f2a3ea02c0d46c61b1ff10fR185`. I think it is better to just throw exception for failing situation.

Reply:
Updated code to throw exception when HTTP request fails for `removeandupdateDataBundle`